index.html

```diff
@@ -61,176 +68,280 @@ <h1 id="idh1">
-                    <li>
-                        <a href="#"><button class="b1">html</button></a>
-                    </li>
-                    <li>
-                        <a href="#"><button class="b2">css</button></a>
-                    </li>
-                    <li>
-                        <a href="#"><button class="b3">javascript</button></a>
-                    </li>
-                </ul>
-                <a href="#"><button class="b4">See Project</button></a>
             </article>
         </section>
-        <section class="about-contact">
-            <article class="about-myself">
-                <h2>About <br> Myself</h2>
-                <p>
-                    Hello I’m a software developer! I can help you build a product , feature or website Look through
-                    some of
-                    my work and experience! If you like what you see and have a project you need coded, don’t hestiate
-                    to
-                    contact me.
-                </p>
-                <div id="socialid">
-                    <h3><a href="#" id="connectid">
-                            LET'S CONNECT
-                        </a></h3>
-                    <ul class="ic">
-                        <li><a href="https://github.com/CarlosIgreda" target="_blank" rel="noopener"><img
-                                    src="./images/Github.svg" alt="Github"></a></li>
-                        <li><a href="https://www.linkedin.com/in/carlos-jes%C3%BAs-igreda-rivas-323763116/"
-                                target="_blank" rel="noopener"><img src="./images/Linkedin.svg" alt="Linkedin"></a></li>
-                        <li><a href="https://www.angellist.com/" target="_blank" rel="noopener"><img
-                                    src="./images/AngelList.svg" alt="AngelList"></a></li>
-                        <li><a href="https://twitter.com/carlosigreda" target="_blank" rel="noopener"><img
-                                    src="./images/Twiter.svg" alt="Twiter"></a></li>
-                        <li><a href="https://medium.com/" target="_blank" rel="noopener"><img src="./images/Medium.svg"
-                                    alt="Medium"></a></li>
-                    </ul>
-                </div>
-                <a href="#"><button class="b5">Get my resume</button></a>
-                <ul class="aboutul1">
-                    <li>
-                        <div id="idlanguage" class="more-info">
-                            <h2>Languages</h2>
-                            <img src="./images/down-arrow.svg" alt="down-arrow">
+        <section class="aboutcontact">
+            <article class="aboutmyself">
+                <div class="gabout">
+                    <div id="div1about">
+                        <h2>About <br> Myself</h2>
+                        <p>
+                            Hello I’m a software developer! I can help you build a product , feature or website Look
```

Comment:
- [ ] Please fix the overflow in all your web pages, if the user goes to the right then everything will go messy you should consider adding an overflow-x: hidden property to your body tag then you can realign all the items so everything looks like the design and responsive as we want.
![image](https://user-images.githubusercontent.com/74744787/222895465-9d0e244a-781f-4043-8902-85bd16ee33bd.png)

Reply:
Thanks for your review @MeqdamAlqudah. I've followed your suggestion and I added an overflow-x: hidden in the body tag and also now is responsive in the desktop version as you mentioned.

Reply:
Please @MeqdamAlqudah, could you check my pull request before sending the code review? Just to ensure that everything is okay. Thanks in advance.